firewall/outgoing_blocker_default.go

```diff
@@ -27,35 +27,35 @@ const (
 )
 
 // DefaultTrackingBlocker traffic blocker bootstrapped for global calls
-var DefaultTrackingBlocker TrafficBlocker = &noopTrafficBlocker{}
+var DefaultTrackingBlocker OutgoingTrafficBlocker = &outgoingBlockerNoop{}
 
-// TrafficBlocker interface neededs to be satisfied by any implementations which provide firewall capabilities, like iptables
-type TrafficBlocker interface {
+// OutgoingTrafficBlocker interface neededs to be satisfied by any implementations which provide firewall capabilities, like iptables
```

Comment:
`OutgoingTrafficBlocker` and `IncomingTrafficBlocker` have the same godoc, but obviously they are 2 different interfaces providing different functionality. I think you could explain what is the purpose of each, and also skip the `interface neededs to be satisfied by any implementations which provide` - because this is true for any interface.